indico/modules/rb/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -161,7 +164,9 @@ class BookingEditForm extends React.Component {
       onBookingPeriodChange,
       formProps,
       hideOptions,
+      onExtraFieldsChange,
     } = this.props;
+    const {booking} = this.props;
```

Comment:
I'd move this to the big unpacking right above, and then this one to a separate statement here?

    const {bookedForUser, startDt, endDt, room, isAccepted, repetition} = booking;